### ISSUE TITLE
Best structures are not being written

### ISSUE THREAD

**Issue by jchodera** [choderalab/covid-moonshot-infra#46]
It seems like the best transformation is not having its associated structures written. We should revisit the criteria for which structures should be written.

It's a little odd right now because we're going from [design we want to evaluate] -> [reference molecules], so a positive number indicates a structure we're interested in right now.

**Comment by jchodera** [comment]
It looks like perhaps just the _best_ structure (the most negative DeltaG) is not being written. In the case of Sprint 3, it's currently `RUN504`, but subsequent good RUNs seem to be written.

**Comment by jchodera** [comment]
I suspect this might be an off-by-one error somewhere.

**Comment by jchodera** [comment]
In particular, the `/home/server/server2/projects/available/covid-moonshot/synthetic-enumeration/2020-08-20-benzotriazoles.json` file has the key index the same as the RUN number:
```JSON
    "0": {
        "JOBID": 0,
        "directory": "RUN0",
        "end": 0,
        "end_pIC50": "5.74624419353312",
        "end_smiles": "CN(C)c1ccc(cc1)N(Cc2ccsc2)C(=O)Cn3c4ccccc4nn3",
        "end_title": "ALP-POS-d2866bdf-1",
        "ff": "openff-1.2.0",
        "ligand": "2020-08-20-benzotriazoles-dockscores-x10876.sdf",
        "protein": "../receptors/monomer/Mpro-x2646_0_bound-protein-thiolate.pdb",
        "start": 1,
        "start_smiles": "CN(C)c1ccc(cc1)N(Cc2ccccc2)C(=O)Nn3c4ccccc4nn3",
        "start_title": "BENZO_1_1",
        "target": "SARS-CoV-2 Mpro"
    },
```
However, when the `automate-plotting` branch code is started, it tries to start with `RUN-1`, which leads me to believe that there's a `-1` being applied somewhere. We should be very careful that this doesn't lead us to select the wrong molecules!
```
WARNING:root:Failed to process run -1: Failed to extract work values for complex: Empty result set for project path '/│   15 root      20   0       0      0      0 S   0.0   0.0   0:00.01 cpuhp/1                                           
home/server/server2/data/SVR314342810/PROJ13424', run -1  
```

**Comment by jchodera** [comment]
The issue is [here](https://github.com/choderalab/covid-moonshot-infra/blob/master/covid_moonshot/core.py#L82):
```python
    def run_id(self) -> int:
        return self.JOBID - 1
```
This is my fault---I had changed the JSON file to sync up the JOBID and RUN index without properly passing on that information. I'll fix this in https://github.com/choderalab/covid-moonshot-infra/pull/34

**Comment by hannahbrucemacdonald** [comment]
@jchodera it seems like this can be closed?

**Comment by jchodera** [comment]
This was fixed in #34.
